Makefile

```diff
@@ -65,8 +66,10 @@ build-test:	build-cluster build-cli
 	cargo build --bin flv-test $(RELEASE_FLAG) $(TARGET_FLAG) $(VERBOSE_FLAG)
 
 install_rustup_target:
-	./build-scripts/install_target.sh
-	
+	rustup target add $(TARGET)
```

Comment:
Why this is removed?

Reply:
The only thing `install_target.sh` did was execute `rustup target add` or `cargo install cross`, but I worked that into the Makefile itself since now we have to distinguish between two targets (arm and armv7)

Reply:
Keep existing script because it is not easy to add complex condtional